Code/Framework/AzCore/AzCore/Asset/AssetManager.cpp

```diff
@@ -1282,6 +1285,11 @@ namespace AZ::Data
         AssetMap::iterator it = m_assets.find(assetId);
         if (it == m_assets.end())
         {
+#if defined(CARBONATED) && defined(ENABLE_ASSET_MEMORY_TRACKING_OVERHEAD)
```

Comment:
Each time I use extra-CPU even if memory tracking is OFF I use the overhead definition fence.